### PR TITLE
Fraoustin/list dsn

### PR DESCRIPTION
**Pull request by fraoustin**
## Description

add option list-dsn : get all alias in aliasdsn section of config file

## Checklist

- [x] I've added this contribution to the `changelog.md`.
- [x] I've added my name to the `AUTHORS` file (or it's already there).

### CODE REVIEW

**Comment by fraoustin** [comment]
Hi i don't understand the failure of Travis. Someone can help me, please

**Comment by tsroten** [comment]
@fraoustin That version of pep8radius has a bug in a dependency, try making these changes:
https://github.com/dbcli/mycli/pull/574/commits/1da113b5491dbf76a5dcd7c452be27e61e91462a

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "restructuredtext.confPath": "c:\\Users\\faoustin\\Downloads\\pgcli"
+}
```
Comment (tsroten) [inline .vscode/settings.json]: Was committing this file an accident? Maybe add `.vscode` to your personal gitignore file?

**Comment by tsroten** [comment]
@fraoustin Nice 👍 👨‍🚀 

Like I mentioned on the mycli pull request, it would be nice to mask or strip the password from the DSN output.

**Comment by j-bennet** [comment]
@fraoustin The problem with tests (autopep8) is fixed on master by https://github.com/dbcli/pgcli/pull/843. You should be good after you rebase. You still need to remove `.vscode/settings.json` from the PR.

**Comment by j-bennet** [comment]
Useful! 👍

**Pull request merged**